src/utils/db.ts

```diff
@@ -22,9 +22,10 @@ export const findOrCreateEstimate = async (
     let currentEstimate = { ...estimate };
 
     if (estimate.id == null) {
-        const { id: _, ...whereParams } = currentEstimate;
+        const { id: _, q = null, date = null, ...whereParams } = currentEstimate;
```

Comment:
Why this is not `estimate`? Do we really need a copy of it?

Reply:
like a copy is not needed, now made without a copy